src/main/java/graphql/nadel/engine/Transformer.java

```diff
@@ -86,6 +85,7 @@
     final ServiceExecutionHooks serviceExecutionHooks;
     private OverallTypeInformation<?> overallTypeInformation;
     private TransformationMetadata transformationMetadata;
+    private Map<String, GraphQLError> isFieldAllowedErrors;
```

Comment:
Seems like this should be `forbiddenFieldsErrors` or something

Side rant, it's always annoying trying to find out what the `String` key is here, and encoding that info in the field name always sucks. Kotlin has this great thing called [inline classes](https://kotlinlang.org/docs/reference/inline-classes.html), and type aliases help too and they're not as strict.